src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -206,7 +203,11 @@ class ArrayField extends React.Component {
     const addAnotherDisabled = items.length >= (schema.maxItems || Infinity);
 
     return (
-      <div className={itemsNeeded ? 'schemaform-review-array-warning' : null}>
+      <div
+        className={`${
+          itemsNeeded ? 'schemaform-review-array-warning' : ''
+        } ${fieldName}`}
```

Comment:
What do you think about adding the `fieldName` to the `name` property? This way, we don't have any field name / class name collision (slim though the chances of that happening are).

Reply:
Sounds like a better solution!